src/Microsoft.Health.Fhir.Liquid.Converter/Models/ProcessorSettings.cs

```diff
@@ -9,5 +9,8 @@ public class ProcessorSettings
     {
         // Time out for rendering templates in milliseconds. By default no time out is set, which is zero in DotLiquid.
         public int TimeOut { get; set; } = 0;
+
+        // Max iterations for rendering templates.
+        public int MaxIterations { get; set; } = 100000;
```

Comment:
It seems that the properties of this setting class are read-only right? Maybe adjusting them to `{ get; }` makes more sense?

Reply:
I feel we should allow the properties be edited, in case users need to temporarily updated them.
Refer to [Newtonsoft.Json/JsonSerializerSettings](https://github.com/JamesNK/Newtonsoft.Json/blob/master/Src/Newtonsoft.Json/JsonSerializerSettings.cs)